### PR TITLE
adds tests for core/span_context.go

### PR DESCRIPTION
**Pull request by djosephsen**

### CODE REVIEW

File: api/core/span_context_test.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	}{
+		{
+			name: "fourtytwo",
+			sc: SpanContext{
+				TraceID: TraceID{
+					High: uint64(42),
+					Low:  uint64(42),
+				},
+			},
+			want: `000..000`,
```
Comment (rghetia) [inline api/core/span_context_test.go]: this should be non-zero.

File: api/core/span_context_test.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+		}, {
+			name: "empty",
+			sc:   SpanContext{TraceID: TraceID{}},
+			want: `000..000`,
+		},
+	} {
+		t.Run(testcase.name, func(t *testing.T) {
+			//proto: func (sc SpanContext) TraceIDString() string {}
+			have := testcase.sc.SpanIDString()
```
Comment (rghetia) [inline api/core/span_context_test.go]: this should testcase.sc.TraceIDString()

**Comment by rghetia** [comment]
@djosephsen please sign CLA.

**Comment by djosephsen** [comment]
I signed the cla back in June..

Screen Shot 2019-07-19 at 1 12 17 PM

**Comment by djosephsen** [comment]
I signed it

**Comment by djosephsen** [comment]
(trying the faq fix) 

> If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.

**Comment by rghetia** [comment]
> (trying the faq fix)
> 
> > If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.

are you using same email?

**Comment by djosephsen** [comment]
*sigh* no.  Should I close this, fix my config and try again with a new PR?

**Pull request closed without merging**